manifests/kustomize/env/cert-manager/platform-agnostic-k8s-native/kustomization.yaml

```
@@ -0,0 +1,58 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - ../../../base/crds
+  - ../../../base/installs/generic
+  - ../../../base/metadata/base
+  - ../../../third-party/argo/installs/namespace
+  - ../../../third-party/minio/base
+  - ../../../third-party/mysql/base
```

Does this work to not duplicate this?
```suggestion
  - ../../platform-agnostic
```